src/ekat/ekat_pack.hpp

```diff
@@ -372,7 +372,7 @@ KOKKOS_INLINE_FUNCTION
 OnlyPack<PackType> pow (const PackType& a, const ScalarType/*&*/ b) {
   PackType s;
   vector_simd for (int i = 0; i < PackType::n; ++i)
-    s[i] = std::pow<typename PackType::scalar>(a[i], b);
+    s[i] = std::pow(a[i], b);
```

Comment:
I'm actually wondering why we had the template argument in the first place. I'm super ok without it (lighter code, beside fixing clang/mac builds). But I'm wondering if the template arg was needed in some configurations. @jgfouca do you have any idea?

Reply:
I was also pretty confused when I saw the explicit use of the template parameter there. I wonder if it's just from an older compiler used by one of the machines.

Reply:
The AT is not complaining on SKX, KNL, and V100, so I'm ok with the removal.